TUM Campus App/CardViewController.swift

```diff
@@ -72,21 +75,40 @@ extension CardViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setupLogo()
+        
+        if #available(iOS 11.0, *) {
+            self.navigationController?.navigationBar.prefersLargeTitles = false
+            self.navigationController?.navigationItem.largeTitleDisplayMode = .never
```

Comment:
shouldn't `largeTitleDisplayMode = .never` be enough? I think you don't have to set `prefersLargeTitles` every time